File: sept22/sept22.rb
# book = {
#   title: 'jaws',
#   page: 5
# }
require 'pry'

book_ruby = {}
string = "title=jaws&page=5"
string = string.split(/\W/)
# ruby
title = string[0]
page = string[2]

book_ruby[:title] = string[1]
book_ruby[:page] = string[3].to_i

puts book_ruby

binding.pry

File: sept22/sept22.js
console.log('working')

// book = {
//   title: 'jaws',
//   page: 5
// }

var string = "title=jaws&page=5"
string = string.split(/\W/)

var book_js = {}
var title = string[0]
var page = string[2]

book_js[title] = string[1]
book_js[page] = Number(string[3])

console.log(book_js)
